DSharpPlus.Test/PermissionInheritanceCommands.cs

```diff
@@ -25,30 +25,29 @@
 using DSharpPlus.CommandsNext;
 using DSharpPlus.CommandsNext.Attributes;
 
-namespace DSharpPlus.Test
+namespace DSharpPlus.Test;
+
+[CannotUse]
+public sealed class PermissionInheritanceCommands : BaseCommandModule
 {
-    [CannotUse]
-    public sealed class PermissionInheritanceCommands : BaseCommandModule
+    [Group("borked")]
+    public sealed class BrokenPermissionCommands : BaseCommandModule
```

Comment:
was this intentional? are we sure this doesnt break anything?

Reply:
upon more consideration, i might just have misread the diff. please double-check whether this is correct, but don't put too much store by this review comment

Reply:
apparently dotnet format did something weird with this file, gonna revert changes.